swap/src/lib.rs

```diff
@@ -1,7 +1,5 @@
 #![warn(
     unused_extern_crates,
-    missing_debug_implementations,
-    missing_copy_implementations,
```

Comment:
Thanks to your change I did a bit of reading: https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#missing-copy-implementations

I am happy eitherway, I think what is important is that if we decide to keep the lint then we must be weary to not put copy on data is likely to be of a huge size.

Consider the domain of this repo and the fact that we may have some huge byte array, removing the lint may actually be the better strategy 👍

Reply:
ok I will remove the missing copy lint

Reply:
> ok I will remove the missing copy lint

It is already removed, no action is necessary for this comment.